modules/salt/pages/formula-monitoring.adoc

```diff
@@ -41,11 +43,22 @@ For more information about using monitoring, see xref:administration:monitoring.
     For example, ``15`` will scrape data every fifteen seconds.
 * In the [guimenu]``Evaluation interval`` field, type the FIXME, in seconds.
```

Comment:
Is FIXME something in the dashboard.  If not, we must rephrase it somehow.

Reply:
Fixed